spec/API_specification/signatures/_sorting_functions.py

```diff
@@ -0,0 +1,43 @@
+def argsort(x, /, *, axis:int = -1, descending:bool = False, stable:bool = True):
```

Comment:
I would not put an underscore in the front of the file name. This isn't a real module that people will import, and the underscores just make typing them harder.

Reply:
I would suggest copying the function stubs from https://github.com/data-apis/array-api-tests/tree/master/array_api_tests/function_stubs. These are already extracted from the spec, and formatted correctly. One thing that is missing here is the return annotation.

Reply:
(and then obviously once this is in we can replace the function stubs in the tests with just literally importing this).

Reply:
I'll copy them here with the `_types.py`